nav2_costmap_2d/src/footprint_subscriber.cpp

```diff
@@ -59,9 +59,10 @@ FootprintSubscriber::getFootprint(
     return false;
   }
 
+  auto current_footprint = std::atomic_load(&footprint_);
```

Comment:
I think you're missing some includes in order to use these functions

Reply:
"#include <memory>" has include the two functions. Ref is [here](https://en.cppreference.com/w/cpp/memory/shared_ptr/atomic). I updated the descriptions. The two functions are supported since c++11, not c++14.

Reply:
OK, I was looking at the non-shared pointer docs that wanted `<atomic>` but sounds good